asciidoc/courses/graphql-basics/course.adoc

```diff
@@ -1,14 +1,11 @@
 = Introduction to Neo4j & GraphQL
 :categories: developer:2, graphql:1
 :caption: Learn how to create GraphQL API using Neo4j GraphQL Toolbox and Library.
-:status: draft
+:status: active
```

Comment:
If we merge with draft status, Neo4j employees will be able to enrol.

Reply:
Yes

I was having trouble in my dev environment getting the course to be open to enrollment, I change this in an attempt to get it to open.

Changed back to draft.